.gitignore

```diff
@@ -0,0 +1,12 @@
+# Temporary files.
+.#*
```

Comment:
Not sure of a specific reason for this?  Would like to know if there is one.

Reply:
Found an answer https://stackoverflow.com/questions/38337833/gitignore-what-is-the-meaning-of-and-these-other-patterns

Reply:
Yeah, as you can imagine I left most of these as they were. For my setup `.#` or `.DS_Store` are not relevant.